app/src/main/res/values/strings.xml

```diff
@@ -154,6 +160,14 @@
     <string name="privacy_contact_address">Piwigo, 42 rue des Vignes, 21800 Quetigny (France)</string>
     <string name="privacy_contact_email" translatable="false">android@piwigo.org</string>
 
+    <!-- Settings -->
+    <string name="settings_title">Settings</string>
+    <string name="settings_albums_title">Albums</string>
+    <string name="settings_thumbnails_size">Thumbnails Size</string>
```

Comment:
should be "Download" instead of Thumbnail, as it is used not only for the thumbnail, but also for the default for the full-screen photo viewer.

Reply:
ok